src/functionBased/components/TodoContainer.js

```diff
@@ -0,0 +1,111 @@
+
+
+import React, { useState, useEffect } from "react"
+import { Switch, Route } from 'react-router-dom';
+import Header from "./Header"
+import InputTodo from "./InputTodo"
+import TodosList from "./TodosList";
+import About from "../pages/About"
+import NotMatch from "../pages/NotMatch"
+import Navbar from "./Navbar";
+import { v4 as uuidv4 } from "uuid";
+
+
+
+const TodoContainer = () => {
+
+  const [todos, setTodos] = useState(getInitialTodos())
+
+  // useEffect(() => {
+  //   // getting stored items
+  //   const temp = localStorage.getItem("todos")
+  //   const loadedTodos = JSON.parse(temp)
+
+  //   if (loadedTodos) {
+  //     setTodos(loadedTodos)
+  //   }
+  // }, [])
```

Comment:
Kindly remove this code and avoid commenting unsed code in your project as it makes your code looks very messy